### DISCUSSION TITLE
Pull metadata from a topic to the map

### DISCUSSION THREAD

**Discussion by gbebie-valerian-antidot** [dita-ot/dita-ot#4508] [Q&A]
Hello,
I need to apply the reverse process by retrieving specific metadata information declared in prologs and copying them in the topicmeta element of the root map. I'm currently working with the extension point **dita.xsl.mappull** to achieve that and I fear information from the topics isn't available at this stage.
So, I'm looking for a posterior extension point like **dita.xsl.html5** but I'm struggling to get info from the topic to make it visible in the map.
Is there any resource I could work from to achieve that action?

**Comment by jelovirt** [comment]
There's no out of the box support for this. I think your best bet is to use `depend.preprocess.post` extension point. There you can run an XSLT transformation on the start map and collect metadata from linked topics.

**Reply by gbebie-valerian-antidot** [reply]
Yes, that was one of my options, using the depend.preprocess.post extension point, but my attempts remained unsuccessful.
